src/plugins/vis_types/timeseries/public/application/components/panel_config/table.tsx

```diff
@@ -91,13 +92,19 @@ export class TablePanelConfig extends Component<
     (name: keyof TimeseriesVisParams) => (e: React.ChangeEvent<HTMLInputElement>) =>
       this.props.onChange({ [name]: e.target.value });
 
+  handleNumberChange =
+    (name: keyof TimeseriesVisParams, isClearable: boolean) =>
+    (e: React.ChangeEvent<HTMLInputElement>) =>
+      this.props.onChange({
+        [name]: isClearable && !e.target.value ? undefined : Number(e.target.value ?? 0),
```

Comment:
I think with your latest changes now we don't need `isClearable`

Reply:
I thought it's good option, because in future we can have fields which should be not clearable. But for now I can remove it